modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step05_common_geom.md

```diff
@@ -0,0 +1,206 @@
+- (Cartesian sibling -- [CartesianConcentricCircleAdaptiveBoundaryMeshGenerator.md])
+
+### Geometry Features
+
+- Hexagonal mesh with assembly metadata
+- Can optionally match other assembly meshes' external boundary to enable stitching to generate core meshes.
+
+### Notes
+
+- It can be used without specifying [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/sides_to_adapt) and [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/meshes_to_adapt_to) to generate [PolygonConcentricCircleMeshGenerator.md]-style hexagonal mesh with assembly mesh metadata. The azimuthal discretization of the sides listed in [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/sides_to_adapt) will be identical to the input meshes [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/meshes_to_adapt_to) and overwrite the appropriate entry(s) in [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/num_sectors_per_side).
+- The size of the hexagon ([!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/hexagon_size)) must match the size of the input meshes. Default style for providing [!param](/Mesh/HexagonConcentricCircleAdaptiveBoundaryMeshGenerator/hexagon_size) is `apothem` (half-pitch).
+
+### Example
+
+!media tutorial04_meshing/base_ex_hccabmg.png
+       id=tutorial04-base_ex_hccabmg
+       caption=A schematic drawing showing the input and generated meshes of [HexagonConcentricCircleAdaptiveBoundaryMeshGenerator.md].
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+
+!listing base_mesh_generators/common_geo.i
+         id=tutorial04-hex_2d.i
+         caption= Reflector-style assembly example input.
+         block=Mesh/adaptive_assm
+
+## Core
+
+The highest level of reactor geometry is generally the core, which consists of multiple assemblies patterned together. [PatternedHexMeshGenerator.md] can be used to create cores similar to how it is used to create assemblies. The main difference is that the inputs to a core object are assemblies rather than pins, and the parameter [!param](/Mesh/PatternedHexMeshGenerator/generate_core_metadata) should be set to `true` when creating a core. This tells MOOSE that the generated object is a core rather than an assembly, which is required to identify eligible mesh operations after creation (for example trimming along a line).
+
+### Object
+
+- [PatternedHexMeshGenerator.md]
+- (Cartesian sibling -- [PatternedCartesianMeshGenerator.md])
+
+### Geometry Features
+
+- 2D regular assembly pattern with uniform assembly pitch
+
+### Notes
+
+- Assembly meshes generated by [PatternedHexMeshGenerator.md], [HexagonConcentricCircleAdaptiveBoundaryMeshGenerator.md] and [SimpleHexagonGenerator.md] can be used as inputs. If [SimpleHexagonGenerator.md] is used, the assemblies are homogenized.
+- Core mesh generation should include the parameter [!param](/Mesh/PatternedHexMeshGenerator/generate_core_metadata) as `true`.
```

Comment:
```suggestion
- Core mesh generation should include the parameter [!param](/Mesh/PatternedHexMeshGenerator/generate_core_metadata) as `true`.
- The input pattern will be automatically rotated 90 degrees counterclockwise into a "vertex up" position unless otherwise specified with [!param](/Mesh/PatternedHexMeshGenerator/rotate_angle)
```